client/src/components/common/BaseArticleResult.tsx

```
@@ -1,11 +1,12 @@
+//@ts-nocheck
```

why is this here

---

typescript throws an error because BaseAclArticle has no journal field 
```
{article.journal && <Journal>{article.journal}</Journal>}
```

---

Added so it ignores error in `BaseArticleResult`

---

Don't keep the subtitle rendering in this base component then, move it out into an abstract `renderSubtitle()` function that is implemented specifically for each dataset

---

Actually just thinking about it more, we should have two entirely separate `Cord19ArticleResult`, `AclArticleResult` classes that take the same props interface

---

Don't bother extending them off one base class, so just define like `ArticleResultProps` in a Common.tsx file or something and import it into both